code/modules/mob/living/silicon/robot/robot_model.dm

```diff
@@ -266,6 +270,7 @@
 		if(!isnull(details[SKIN_ICON]))
 			cyborg.icon = details[SKIN_ICON]
 			cyborg_icon_override = details[SKIN_ICON] // SKYRAT EDIT ADDITION
+		if(!isnull(details[SKIN_PIXEL_X]))
```

Comment:
either needs to be done modularly or marked as a skyrat edit

Reply:
TG code